src/Core/Content/Seo/SalesChannel/SeoResolverData.php

```diff
@@ -38,7 +49,23 @@ public function getIds(string $entityName): array
         return array_keys($this->entityMap[$entityName]);
     }
 
+    /**
+     * @deprecated tag:v6.8.0 - use getAll instead
+     */
     public function get(string $entityName, string $id): Entity
```

Comment:
please add a changelog describing the deprecation and add a `# next major version` entry describing the break and what to do instead